pkg/buildscript/mutations.go

```diff
@@ -33,7 +35,11 @@ func (b *BuildScript) UpdateRequirement(operation types.Operation, requirement t
 	return nil
 }
 
-func (b *BuildScript) addRequirement(requirement types.Requirement) error {
+func (b *BuildScript) AddRequirement(requirement types.Requirement) error {
```

Comment:
By changing the visibility of this method, it's possible that `UpdateRequirement()` is no longer needed. Please consider removing it.

Reply:
Yeah that was my expectation, but `state import` and buildscript merging still makes good use of it.